manage.py

```diff
@@ -119,11 +119,16 @@ def init_test_db(force=False):
 
 @cli.command()
 @click.argument("archive", type=click.Path(exists=True))
-def import_data(archive):
+@click.option("--is-dataset-dump", "-d", is_flag=True, help="Import dataset dumps.")
```

Comment:
Would adding a new command for importing dataset dumps (like the new command for creating dataset dumps) make more sense?